run_tests.sh

```diff
@@ -4,25 +4,22 @@
 # ./run_tests.sh                         # local, go 1.12
 # ./run_tests.sh docker                  # docker, go 1.12
 # ./run_tests.sh podman                  # podman, go 1.12
+#
+# To use build tags:
+#  TESTTAGS="chartdata" ./run_tests.sh
+#  TESTTAGS="pgonline" ./run_tests.sh
+#  TESTTAGS="pgonline fullpgdb" ./run_tests.sh
```

Comment:
Note this tag convention is used with `go test`. e.g. `go test . -v -tags "pgonline fullpgdb" -count 1`